### ISSUE TITLE
Angular: Changing window size in CodeSandbox crashes Handsontable

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/examples#2]
### Description
 Changing window size in CodeSandbox crashes Handsontable

### Steps to reproduce

1. Resize the example window

![angular](https://user-images.githubusercontent.com/10757813/74439633-e6923f00-4e6c-11ea-9660-39861d2ddaa9.gif)

### Demo

https://codesandbox.io/s/my-demo-uorsc Angular demo broken 😢 
https://codesandbox.io/s/date-dependent-time-change-jh9s3 Vanilla version working 👍 
https://codesandbox.io/s/react-ok-hn71u React version working 👍 
https://codesandbox.io/s/vue-ok-2cc54 Vue version working 👍 

### Your environment
* Angular version: 8.2.14
* Handsontable version: 7.3.0
* Wrapper version: 5.1.0
* Browser Name and version: Chrome 79
* Operating System: Mac Catalina

Transfered from https://github.com/handsontable/angular-handsontable/issues/194

**Comment by wojciechczerniak** [comment]
Hmm, we don't have Angular example in this repository 🤔

**Comment by AMBudnik** [comment]
If there's any better place feel free to pass it back. It has already been transferred from the Angular repo.

**Comment by mrpiotr-dev** [comment]
Ok, I know what was happened:
This is v5.0.1 (the last one, currently - correctly working, version :scream:):
![obraz](https://user-images.githubusercontent.com/11849621/74492319-d743dd80-4ece-11ea-8ec6-9b399fc20568.png)

and in v5.0.2

![obraz](https://user-images.githubusercontent.com/11849621/74492475-44577300-4ecf-11ea-955d-dcb8fa3e599b.png)

As you can see `undefined != null // => false` but  `undefined !== null // => true`.

I've introduced this "feature" during ESLINT-related improvements 🤦‍♂. Perhaps `no-reassign` or so.
I guess that was one of these PRs with billions of changes - impossible to check line by line.
The point is, `window` doesn't inherit from `Node` and has no `parentNode` property.

Fortunately, it's trivial to fix 🤓

BUT, there is even something worse than that!
You're probably wondering `Whaaaat?` and  `Why react/vue wrappers and vanilla version work well?`

The answer is here: https://github.com/handsontable/handsontable/blob/7.4.0/src/helpers/feature.js#L71-L75
and here: https://developer.mozilla.org/en-US/docs/Web/API/Element/createShadowRoot

TL;DR
`Element.prototype.createShadowRoot` is now `undefined` - that means Polymer, AngularJS and Angular (all of them using custom node name -> `HOT-TABLE`) blew up after the update to the Chromium v80.
Read more here: https://developers.google.com/web/updates/2019/12/chrome-80-deps-rems#web_components_v0_removed
And https://bugs.chromium.org/p/chromium/issues/detail?id=1048923#c7

That means, we have the fire on board 😱    🔥🚒🔥

**Comment by mrpiotr-dev** [comment]
Same as in https://github.com/handsontable/angular-handsontable/issues/193 - works with 7.4.2 🙂

**Comment by AMBudnik** [comment]
Cool! Thanks for checking @swistach
